NeoDebuggerUI/ViewModels/MainWindowViewModel.cs

```diff
@@ -101,7 +101,7 @@ private bool LoadContract(string avmFilePath)
                 if (File.Exists(cSharpFile))
                 {
                     SelectedFile = cSharpFile;
-                    AddBreakpoints(); // test
+                    //AddBreakpoints(); // test
```

Comment:
All commented code must have a comment above explaining why it is commented, if you are planning to change it later you should put a // TODO: <imperative phrase>